### DISCUSSION TITLE
v1.319 🌈

### DISCUSSION THREAD

**Discussion by bitwiseman** [hub4j/github-api#1797] [General]
## Changes

- Prepare release (bitwiseman): github-api-1.319 @github-actions (#1795)
- Chore(deps): Bump com.diffplug.spotless:spotless-maven-plugin from 2.27.2 to 2.43.0 @dependabot (#1785)
- Add withName method to GHCheckRunBuilder @halkey (#1769)
- Chore(deps): Bump actions/upload-artifact from 3 to 4 @dependabot (#1774)
- Support state reason for issues @stianst (#1793)
- Allow to list workflow runs by head sha and created date @gsmet (#1789)
- Chore(deps): Bump com.squareup.okio:okio from 3.5.0 to 3.7.0 @dependabot (#1777)
- Chore(deps): Bump org.apache.bcel:bcel from 6.8.0 to 6.8.1 @dependabot (#1784)
- Chore(deps): Bump codecov/codecov-action from 3.1.4 to 4.0.0 @dependabot (#1786)
- Fix `GHFileNotFoundException` when getting commits from PR found in search @NlightNFotis (#1779)
- Add payload for discussion_comment event @gsmet (#1781)
- Chore(deps): Bump github/codeql-action from 2 to 3 @dependabot (#1773)
- Chore(deps): Bump org.apache.maven.plugins:maven-surefire-plugin from 3.2.2 to 3.2.3 @dependabot (#1776)
- Chore(deps): Bump org.apache.bcel:bcel from 6.7.0 to 6.8.0 @dependabot (#1775)
- Chore(deps): Bump org.apache.maven.plugins:maven-project-info-reports-plugin from 3.4.2 to 3.5.0 @dependabot (#1766)
- Chore(deps): Bump com.github.spotbugs:spotbugs-maven-plugin from 4.7.3.0 to 4.8.1.0 @dependabot (#1763)
- Chore(deps): Bump org.apache.bcel:bcel from 6.6.1 to 6.7.0 @dependabot (#1764)
- Chore(deps): Bump actions/setup-java from 3 to 4 @dependabot (#1762)
- Chore(deps): Bump org.jacoco:jacoco-maven-plugin from 0.8.10 to 0.8.11 @dependabot (#1761)
- Chore(deps-dev): Bump com.google.code.gson:gson from 2.10 to 2.10.1 @dependabot (#1760)
- Chore(deps-dev): Bump com.tngtech.archunit:archunit from 1.1.0 to 1.2.0 @dependabot (#1759)
- Update pom.xml test executions with artifactId  config @bitwiseman (#1754)
- Add MANNEQUIN to GHCommentAuthorAssociation enum  @choichanhyeok (#1753)
- Prepare release (bitwiseman): github-api-1.318 @github-actions (#1751)

<hr /><em>This discussion was created from the release <a href='https://github.com/hub4j/github-api/releases/tag/github-api-1.319'>v1.319 🌈</a>.</em>

**Comment by bitwiseman** [comment]
@gsmet  
https://github.com/hub4j/github-api/actions/runs/7986594531 - The release job failed, but it looks like the build was still published.  
If this version doesn't work for you, please say so and I'll take another swing at it.

**Reply by gsmet** [reply]
@bitwiseman the release looks good from what I can see. Usually when it fails at this stage, the Sonatype servers are running things as background job so it's no problem. I'm in the process of releasing Quarkus GitHub API 1.319.0, will let you know once I'm completely confident.

**Reply by gsmet** [reply]
@bitwiseman everything seems OK except for the Javadoc published here https://github-api.kohsuke.org/apidocs/index.html that still points to 1.318.

**Comment by bitwiseman** [comment]
@gsmet 
Thanks, I've updated the docs.

**Reply by gsmet** [reply]
Perfect, thanks!
